db/migrate/20180313113654_change_digests_to_bigint.rb

```diff
@@ -0,0 +1,33 @@
+class ChangeDigestsToBigint < ActiveRecord::Migration[5.1]
+  def up
+    # messages digest
+    remove_index :messages, :digest
+    remove_column :messages, :digest
+    add_column :messages, :digest, :bigint, :limit => 8
+    Message.unscoped.all.find_each {|rec| rec.update_attribute :digest, Message.make_digest(rec.value)}
```

Comment:
please in batches!!!

Reply:
Sorry what? It's using `find_each` which defaults to batch of size 1000 records.